10_Examples/jett-onboarding/src/talking.h

```diff
@@ -0,0 +1,23 @@
+#ifndef TALKING_H_
+#define TALKING_H_
+
+#include <iostream>
+#include "string.h"
+
+using namespace std;
+
+class talking
```

Comment:
The most popular convention for naming variables, classes, etc, is gonna be [Google Style Guide:](https://google.github.io/styleguide/cppguide.html).

Its a good read, but im gonna cover main points here for C/C++:
- classes are typically "PascalCase".
- functions/methods are typically "camelCase"
- variables are "snake_case"
- members variables of a class should have an underscore at the end (or sometimes a ",_" prefix.

None of these rules will break your code or change functionality, but its very standard and so helps people understand what they are looking at. when they use your code